src/utils/balances.js

```diff
@@ -38,12 +44,21 @@ export const getTotalBalance = (entries: { [key: string]: BigNumber}): BigNumber
 };
 
 export const getTotalCategoryBalance = (
-  accountTotals: ?CategoryTotalBalancesPerChain,
+  accountTotalBalances: ?CategoryTotalBalancesPerChain,
   category: string,
 ): BigNumber => {
-  const balancesOnChains = (Object.values(accountTotals || {}): any);
+  const balancesOnChains = (Object.values(accountTotalBalances || {}): any);
 
   return sum(balancesOnChains.map((chainTotals) => {
     return chainTotals?.[category] || BigNumber(0);
   }));
 };
+
+// TODO: add return AssetsBalancesPerChain later and fix Flow
+export const getChainAssetsBalancesForCategory = (
```

Comment:
Having one function to get different categories which are of different types will not work with flow. IMO we need to have 1 function per asset category to get any reasonable type checking by flow.